### DISCUSSION TITLE
Coordinator failure - failsafe fallback solution

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#13751] [General]
Recently I was thinking about what happens if my system built upon OpenHAB and Zigbee2Mqtt fails... Obviously, my home is gonna be dark and hostile place (dark due to malfunction of lights and hostile due to anger of my wife). :-) 

I was thinking whether this issue can be somehow mitigated? My current backup solution uses old-fashioned wall switches mounted behind smart ones, so that it is possible to do a power cycle for bulbs and turn them on in case of a disaster.

However, this already requires some activity, which includes some minor disassembly activity on the walls. Is there any way how to setup the network (some binding magic), so that outage of coordinator (e.g. Raspberry failure, ...) would allow the bulbs to respond to zigbee switches? At least the on/off events or so? 

I understand, that it should be possible to have a second system with second coordinator, which would clone configs from z2m, ohab, mqtt, do a handover... But this is quite advanced. Plus, I cannot believe that a zigbee network in principle would have such a single point of failure? 

Any ideas and opinions are welcome.

**Comment by DubhAd** [comment]
I'd suggest:

- Buy a spare coordinator and invest in better hardware than a Pi (or at least, clone the SD card so you can recover easily _when_ it dies)
- Use smart _switches_ rather than smart bulbs. Those will work just fine when the mesh is down.

**Comment by jerrm** [comment]
> I cannot believe that a zigbee network in principle would have such a single point of failure?

Believe it.  Only one coordinator per net AFAIK.  Supposedly one of the things Thread will correct, but we'll have to see how that plays out in real world implementations.

Seems like a powered coordinator stick on an otherwise the dead/locked PC would prevent any sort of automatic scripting unless you had a method to physically cut power to the the injured pc.
